soteria/lib/soteria_std/list.ml

```diff
@@ -16,9 +22,11 @@ let take_count n l =
   let result = aux n l in
   (result, !taken)
 
+(** Combine two lists into a list of pairs. *)
 let combine_opt l1 l2 =
   try Some (combine l1 l2) with Invalid_argument _ -> None
 
+(** Partition a list of results into successful values and errors. *)
```

Comment:
This is not what the function does I believe. It was correct in your previous commits though